### ISSUE TITLE
add a standalone script

### ISSUE THREAD

**Issue by garfieldnate** [garfieldnate/Algorithm-AM#37]
Add a standalone script for running AM classification on data sets:

    --train/--data/--exemplars: training set
    --test: test set; leave-one-out if ommitted
    --project: classic AM project
    --{gangs/summary/etc}: print given report type

**Comment by garfieldnate** [comment]
Done via 66493bf8988655a2940e0d4fb50cbf0ecf5fbbce.
